properties/P000130.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: P000130
+slug: strongly-locally-compact
+name: Strongly Locally Compact
+refs:
+  - doi: 10.2307/24340250
+    name: What is Locally Compact?
+---
+A space $X$ is strongly locally compact if every point in $X$ has a basis of
+compact neighborhoods.
+
+See {{doi:10.2307/24340250}} for discussion of strong vs. weak local compactness.
```

Comment:
Does that doi work?  doi.org does not seem to know about it.

Reply:
That's what's listed at https://www.jstor.org/stable/24340250 but you're right that it's broken. I submitted a report to DOI.org and JSTOR. The paper is not included in MathSciNet.

The PR would definitely be improved if there was another paper taking a look at various kinds of local compactness, but that's what I've found.